lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -34,6 +34,12 @@
 from .serial_console import SerialConsole
 
 
+class _HostCapabilities:
```

Comment:
Refer to other classes to add decorators of dataclass and dataclass_json. So it can be used in runbook to force run on specific settings.